var/spack/repos/builtin/packages/hpctoolkit/package.py

```diff
@@ -185,6 +191,10 @@ class Hpctoolkit(AutotoolsPackage, MesonPackage):
     depends_on("zlib-api")
     depends_on("zlib+shared", when="^[virtuals=zlib-api] zlib")
 
+    depends_on("py-docutils", type="build", when="@develop")
```

Comment:
Is this correct, or should it be `when="+docs"`?

Reply:
This is intended, this PR ensures manpages are unconditionally available which requires `py-docutils`. `py-sphinx` and `py-myst-parser` are only needed to build the manual, so those are `when="+docs"` (+ `when="@develop"` from the variant itself).

`@:2024.01` do not require external dependencies for either manpages or the manual, so this PR primarily re-achieves parity between `@develop` and the full releases.